### PR TITLE
skipper: fix TestDataClients

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
* fix swarm init error handling
* remove TestDataClients invalid swarm options

### CODE REVIEW

File: skipper_test.go [hunk 1]
```diff
@@ -1026,21 +1026,23 @@ func TestDataClients(t *testing.T) {
 	sigs := make(chan os.Signal, 1)
 	go run(o, sigs, nil)
 
-	// wait for proxy being ready
-	for {
+	for i := 0; i < 10; i++ {
```
Comment (AlexanderYastrebov) [inline skipper_test.go:1029]: If we remove `EnableSwarm` then proxy is never ready due to infinite wait

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
Piplieline failed due to #2386

**Comment by szuecs** [comment]
:+1:

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**